SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -211,6 +211,7 @@ void USpatialNetDriver::OnConnectedToSpatialOS()
 
 	if (IsServer())
 	{
+		CreateServerWorkerEntity();
```

Comment:
Maybe only worth doing with multiple server-workers, serves no purpose on the authoritative worker? Arguably could be confusing if it doesn't exist (though I'm not sure why since it shouldn't be used by users)

Reply:
I'd like to keep it for consistency.